app/models/miq_server/worker_management/monitor/quiesce.rb

```diff
@@ -24,7 +24,6 @@ def workers_quiesced?
       return true
     end
 
-    kill_timed_out_worker_quiesce
```

Comment:
Note, you'll see a similar block of code `miq_workers.each...` on line 18 above as in the deleted method.  What I'm suggesting is the `quiesce_workers_loop_timeout?` on line 16 above will handle the timeout, making the deleted `kill_timed_out_worker_quiesce` unnecessary.